frontend/app/src/data/defaults.ts

```diff
@@ -8,6 +8,7 @@ export class Defaults {
   static FLOATING_PRECISION = 2;
   static RPC_ENDPOINT = 'http://localhost:8545';
   static KSM_RPC_ENDPOINT = 'http://localhost:9933';
+  static DOT_RPC_ENDPOINT = 'http://localhost:9933'
```

Comment:
Here it's missing a `;`

Reply:
Also I don't know how this work but having the same port seems to be an issue

Reply:
Let me find out if they have different ports